sklearn/discriminant_analysis.py

```diff
@@ -143,9 +143,6 @@ class LinearDiscriminantAnalysis(BaseEstimator, LinearClassifierMixin,
     .. versionadded:: 0.17
        *LinearDiscriminantAnalysis*.
 
-    .. versionchanged:: 0.17
-       Deprecated :class:`lda.LDA` have been moved to :class:`LinearDiscriminantAnalysis`.
```

Comment:
It printed deprecation warning in the documentation of "LinearDiscriminantAnalysis" instead of lda.LDA.
If you agree with the way I changed lda.LDA and qda.QDA (and the related test), I think we should remove that. Otherwise, I don't know.